android/library/src/test/java/io/logto/android/auth/browser/BrowserSignOutFlowTest.kt

```diff
@@ -106,8 +107,9 @@ class BrowserSignOutFlowTest {
 
         browserSignOutFlow.handleRedirectUri(invalidUri)
 
-        verify(onCompleteMock).invoke(logtoExceptionCaptor.capture())
-        assertThat(logtoExceptionCaptor.firstValue).hasMessageThat()
+        verify { onCompleteMock.invoke(capture(logtoExceptionSlot)) }
```

Comment:
In this test, you presume the `onCompleteMock` will be invoked with a non-null value.
We should not presume any result which will return by the SUT, because we don't know what will happen to our SUT, so we test.